### PR TITLE
Ubuntu managed files integration tests5

### PR DESCRIPTION
**Pull request by tgoettlicher**

### CODE REVIEW

File: lib/system.rb [hunk 1]
```diff
@@ -102,38 +102,23 @@ def run_script(*args)
...
 
-  # Runs the given script on the inspected machine asynchronously and calls the callback method
+  # Runs the given command on the inspected machine asynchronously and calls the callback method
   # periodically with new output when it occurs.
   #
   # Example:
   #
   #     count = 0
-  #     raw_list = run_script_with_progress("changed_managed_files.sh") do |chunk|
+  #     raw_list = run_command_with_progress("time_consuming_command") do |chunk|
```
Comment (thardeck) [inline lib/system.rb]: Why do you change the comments? The method underneath still has the old name.

**Comment by thardeck** [comment]
lgtm besides my comment

File: lib/system.rb (hunk 1 above)
Comment (tgoettlicher) [inline lib/system.rb]: `changed_managed_files.sh` implies that it's a script which is injected. In this example it's about commands in general.

**Comment by thardeck** [comment]
lgtm

**Pull request merged**